README.md

```diff
@@ -182,7 +182,7 @@ module.exports = function(options) {
       return next();
     }
 
-    app.models.UserModel.findById(req.accessToken.userId, function(err, user) {
+    app.models.YourModel.findById(req.accessToken.userId, function(err, user) {
```

Comment:
This is intentionally using a different model name than the remote method below. We want to load the User instance here, while the remote method below can be added to any application model.

Can you please revert this line?